projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-results/search-ai-results.component.html

```diff
@@ -111,3 +156,19 @@
   <div class="adf-skeleton"></div>
   <div class="adf-skeleton adf-skeleton-half"></div>
 </ng-template>
+
+<ng-template #editButtonTemplate let-toggleIcon="toggleIcon" let-disabled="disabled">
+  <button
+    *ngIf="(toggleIcon?.isToggled && !disabled) || (!toggleIcon?.isToggled && disabled)"
+    (click)="setEditing(true)"
+    [disabled]="disabled"
+    mat-flat-button
+    [attr.aria-label]="'CORE.METADATA.ACCESSIBILITY.EDIT' | translate"
```

Comment:
We should not use translations from metadata here as it's not metadata. For metadata we have different place in application. Please add separate translation key or reuse different if there is some